runtime/themes/kanagawa.toml

```diff
@@ -7,109 +7,110 @@
 # because of some theming differences, it's not an exact copy of the original.
 
 ## User interface
-"ui.selection" = { bg = "waveBlue2" }
+"ui.selection"         = { bg = "waveBlue2" }
 "ui.selection.primary" = { bg = "waveBlue1" }
-"ui.background" = { fg = "fujiWhite", bg = "sumiInk1" }
+"ui.background"        = { fg = "fujiWhite", bg = "sumiInk1" }
 
-"ui.linenr" = { fg = "sumiInk4" }
+"ui.linenr"          = { fg = "sumiInk4" }
 "ui.linenr.selected" = { fg = "roninYellow" }
 
-"ui.virtual" = "sumiInk4"
-"ui.virtual.ruler" = { bg = "sumiInk2" }
-"ui.virtual.inlay-hint" = "fujiGray"
+"ui.virtual"                      = "sumiInk4"
+"ui.virtual.ruler"                = { bg = "sumiInk2" }
+"ui.virtual.inlay-hint"           = "fujiGray"
 "ui.virtual.inlay-hint.parameter" = { fg = "carpYellow", modifiers = ["dim"] }
-"ui.virtual.inlay-hint.type" = { fg = "waveAqua2", modifiers = ["dim"] }
+"ui.virtual.inlay-hint.type"      = { fg = "waveAqua2", modifiers = ["dim"] }
+"ui.virtual.jump-label"           = { fg = "peachRed", bg = "sumiInk0", modifiers = ["bold"] }
```

Comment:
I wouldn't set a background color. Setting background colors for text can mess with some ui components like cursorline. I don't believe other themes set the background color for jump labels

Reply:
Thanks for your suggestion @pascalkuthe. I wasn't sure about the background either.
The official theme does use one, but it also change the background of the whole background...
I agree, changing the background of the jump-label without changing the whole background is irrelevant.

I pushed a new commit to remove it